src/main/proto/netflix/titus/titus_job_api.proto

```diff
@@ -71,6 +71,25 @@ message Image {
     string digest = 3;
 }
 
+message NetworkConfiguration {
+    enum NetworkMode {
+        // Unknown, the backend will have to chose a sane default base on other inputs
+        UNKNOWN = 0;
+        // Provides no network access to a container (Not supported yet)
+        NONE = 1;
+        // IPv4 only is a sane default for 2001, where task gets a single IPv4 address only
+        IPV4ONLY = 2;
+        // IPv6 And IPv4 (True Dual Stack)
+        IPV6ANDIPV4 = 3;
+        // IPv6 and IPv4 Fallback uses the Titus IPv4 "transition mechanism" to give ipv4
+        // connectivity transparently without providing every container their own IPv4 address
+        IPV6ANDIPV4FALLBACK = 4;
```

Comment:
Bikeshedding some more, using underscores results in better CamelCasing in languages, and I think may be easier to read.

Reply:
One other thing is you can use proto extensions to customize how the names appear in other configuration (like making it ipv6AndIpv4Fallback), so dont worry too much on making the enum name here match the places where you set it.